modules/overrides/interface.go

```diff
@@ -18,6 +18,8 @@ type Service interface {
 	Interface
 }
 
+type Level string
```

Comment:
I didn't see this used, did I miss it?

Reply:
It was used in a previous iteration of the API, nice catch!